pycontw/pycontw-frontend
# feat(reg/tickets): update content to 2024 in ticket page

## Types of changes


* **New feature**


## Description

Release 2024 Ticket Info 

## Steps to Test This Pull Request

1. Go to “http://localhost:3000/2024/zh-hant/registration/tickets"
2. Check if content has been updated and matched the document: https://docs.google.com/document/d/1fqyk_UEdX1k44xjHzUGevWDQ8PgsIQ9_T32DmTQ_bRw/edit.
3. Repeat step 1-2 for english version “http://localhost:3000/2024/en-us/registration/tickets”

## Related Issue

#523

## Screenshot
![image](https://github.com/pycontw/pycontw-frontend/assets/26858727/b6d277e4-8639-4058-9878-b8db1eca4cb7)

![image](https://github.com/pycontw/pycontw-frontend/assets/26858727/39bbc14c-97c7-48d8-983c-4a7ec2d0129c)
